src/CMakeLists.txt

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
has set `CMAKE_CXX_STANDARD` and `CMAKE_CXX_STANDARD_REQUIRED` in the CMakeLists.txt of base dir.
https://github.com/4paradigm/OpenMLDB/blob/main/CMakeLists.txt#L82-L83